src/Kiota.Builder/Writers/Python/CodeMethodWriter.cs

```diff
@@ -79,7 +79,7 @@ public override void WriteCodeElement(CodeMethod codeElement, LanguageWriter wri
                 WriteDefaultMethodBody(codeElement, writer, returnType);
                 break;
         }
-        writer.DecreaseIndent();
+        writer.CloseBlock(string.Empty);
         writer.WriteLine();
```

Comment:
Should we also remove the `writer. WriteLine();` on line 83 as `writer.CloseBlock(string. Empty);` implicitly calls `WriteLine` ?

Reply:
Makes sense.